### PR TITLE
Basic error logging

### PR DESCRIPTION
**Pull request by dbampalikis**
Add basic logging to most important functions of htsget-server

### CODE REVIEW

File: internal/htsconfig/configuration.go [hunk 1]
```diff
@@ -12,9 +12,10 @@ import (
 	"reflect"
 
 	"github.com/ga4gh/htsget-refserver/internal/htsconstants"
-
 	"github.com/ga4gh/htsget-refserver/internal/htsutils"
 
+	log "github.com/sirupsen/logrus"
+
 	"github.com/getlantern/deepcopy"
```
Comment (kostas-kou) [inline internal/htsconfig/configuration.go]:
```suggestion
	"github.com/ga4gh/htsget-refserver/internal/htsutils"
        "github.com/getlantern/deepcopy"
        
	log "github.com/sirupsen/logrus"
```

File: internal/htsdao/daofactory.go [hunk 2]
```diff
@@ -4,11 +4,13 @@ import (
 	"github.com/ga4gh/htsget-refserver/internal/htsconfig"
 	"github.com/ga4gh/htsget-refserver/internal/htsrequest"
 	"github.com/ga4gh/htsget-refserver/internal/htsutils"
+	log "github.com/sirupsen/logrus"
```
Comment (kostas-kou) [inline internal/htsdao/daofactory.go]:
```suggestion
	"github.com/ga4gh/htsget-refserver/internal/htsutils"
	
	log "github.com/sirupsen/logrus"
```

File: internal/htsserver/filebytes.go [hunk 3]
```diff
@@ -8,6 +8,7 @@ import (
 	"os"
 
 	"github.com/ga4gh/htsget-refserver/internal/htsconstants"
+	log "github.com/sirupsen/logrus"
 
 	"github.com/ga4gh/htsget-refserver/internal/htsutils"
```
Comment (kostas-kou) [inline internal/htsserver/filebytes.go]:
```suggestion
	"github.com/ga4gh/htsget-refserver/internal/htsconstants"
	"github.com/ga4gh/htsget-refserver/internal/htsutils"
	
	log "github.com/sirupsen/logrus"
```

File: internal/htsserver/getreadsdata.go [hunk 4]
```diff
@@ -11,9 +11,11 @@ import (
 	"github.com/ga4gh/htsget-refserver/internal/htsconstants"
 	"github.com/ga4gh/htsget-refserver/internal/htsrequest"
 	"github.com/google/uuid"
+	log "github.com/sirupsen/logrus"
```
Comment (kostas-kou) [inline internal/htsserver/getreadsdata.go]:
```suggestion
	"github.com/google/uuid"
	
	log "github.com/sirupsen/logrus"
```

File: internal/htsserver/getvariantsdata.go [hunk 5]
```diff
@@ -4,6 +4,7 @@ import (
 	"net/http"
 
 	"github.com/ga4gh/htsget-refserver/internal/htscli"
+	log "github.com/sirupsen/logrus"
 
 	"github.com/ga4gh/htsget-refserver/internal/htsconfig"
 	"github.com/ga4gh/htsget-refserver/internal/htsconstants"
```
Comment (kostas-kou) [inline internal/htsserver/getvariantsdata.go]:
```suggestion
	"github.com/ga4gh/htsget-refserver/internal/htscli"
	"github.com/ga4gh/htsget-refserver/internal/htsconfig"
	"github.com/ga4gh/htsget-refserver/internal/htsconstants"
	
	log "github.com/sirupsen/logrus"
```

File: internal/htsserver/requesthandler.go [hunk 6]
```diff
@@ -5,6 +5,7 @@ import (
 
 	"github.com/ga4gh/htsget-refserver/internal/htsconstants"
 	"github.com/ga4gh/htsget-refserver/internal/htsrequest"
+	log "github.com/sirupsen/logrus"
```
Comment (kostas-kou) [inline internal/htsserver/requesthandler.go]:
```suggestion
	"github.com/ga4gh/htsget-refserver/internal/htsrequest"
	
	log "github.com/sirupsen/logrus"
```

**Review by kostas-kou — APPROVED** [review]
Just some minor comments for consistency.
Otherwise it looks good to me

**Review by nanjiangshu — APPROVED** [review]

**Pull request merged**